web/package.json

```diff
@@ -18,55 +18,60 @@
   },
   "dependencies": {
     "@intlify/vite-plugin-vue-i18n": "^6.0.3",
-    "@kyvg/vue3-notification": "^2.4.1",
+    "@kyvg/vue3-notification": "^2.6.1",
     "@meforma/vue-toaster": "^1.3.0",
```

Comment:
@6543 We don't use `@meforma/vue-toaster` at all as far as I can tell. So we can remove it and get rid of the problematic dependency completely.

Reply:
hmm it was added in 58838f225c8425436b83b4aff98a33ad41218e61

Reply:
I guess it was just a test and was never removed again 🙈 

If you search the code for `vue-toaster` it returns just the lines in `package.json` and `pnpm-lock.yaml`

Reply:
I'll have a look into our v015 release branch, we might clean and bump that too ... (#1266)

Reply:
As it wasn't even imported that doesn't matter.